components/PaymentForm.js

```diff
@@ -137,6 +141,11 @@ function PaymentForm({ onClose = () => null }) {
             />
           </div>
         </div>
+        <div className="mt-0 mb-6">
+          <span className="text-xs text-gray-500 mb-1 block">
+            <b>#</b>2.36% of the donation is charged as processing fee, if paid through Debit/Credit Cards
+          </span>
+        </div>
```

Comment:
You can just use `p` here 
```js
<p className="mt-2 text-xs text-gray-600 text-center">
  <strong>*2.36%</strong> of the donation is charged as processing fee, if paid
  through Debit/Credit Cards
</p>
```

**And I think this text looks good below buttons instead of above. @duttakapil What you think?**
![Screenshot 2019-08-20 at 11 42 05 PM](https://user-images.githubusercontent.com/8843216/63372668-330c2e80-c3a4-11e9-9411-22e908ad730f.png)

Reply:
We are using * for marking the fields as required. That is the reason why I used # to clear the ambiguity.

Reply:
oh yeah, think it's okay to use `*` here also or else maybe we can use `**` here.

Reply:
Yes, looks better below the button. Also, I think we should make the text more concise. Something like "2.36% is charged as processing fee for Debit/Credit Cards" should be fine